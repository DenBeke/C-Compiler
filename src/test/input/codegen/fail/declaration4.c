// Declared in other scope

#include <stdio.h>

int main() {

    int a = 5;
    int b = 3;
    
    if(1) {
        int c = a + b;
    }
    
    printf("%d", c);

}