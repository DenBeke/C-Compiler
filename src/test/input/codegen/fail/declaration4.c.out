[31m[ERROR] line 14: Use of undeclared 'c'[0m
